sandstone/app.py

```diff
@@ -14,14 +14,17 @@
 from sandstone import settings
 from sandstone.lib import ui_methods
 from sandstone.lib.app_loader import get_installed_app_static_specs
+from sandstone.lib.handlers.broadcast import BroadcastManager
 import sandstone.urls
 from sandstone.urls import URL_SCHEMA
 
 
 
-class SandstoneApplication(tornado.web.Application):
 
+class SandstoneApplication(tornado.web.Application):
+    broadcast_manager = None
```

Comment:
The `BroadcastManager` instance you're storing here will never be accessed independently of a `SandstoneApplication` instance, so this should not be a static variable. This should be removed, and line 27 left the same.